src/pages/api/auth/[...nextauth].ts

```diff
@@ -16,6 +16,7 @@ export const authOptions = {
       },
     }),
   ],
+  secret: process.env.NEXTAUTH_SECRET,
```

Comment:
What is this secret for?

Reply:
It's the secret needed to encrypt the next auth JWT to work properly with the Spotify Auth. I need to update the readme file.

Reply:
Let's keep that in a separate PR, then :)